hubtty/view/change.py

```diff
@@ -1023,13 +1020,11 @@ def editCommitMessage(self):
                        min_width=60, min_height=20)
 
     def doEditCommitMessage(self, dialog):
-        commit_key = None
```

Comment:
Basically the same for the rest of the file, the deletions are a result of commented out code that has not yet been ported to github.